docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,217 @@
+
+```sh
+testcafe all /tests/sample-fixture.js -r my-reporter
+```
+
+A generated test report will be displayed in the command prompt window. 
+
+If you need to save a report to an external file, you can redirect the command output stream to a file  
+by using the > redirection operator and specifying a report file path.
+
+```sh
+testcafe all /tests/sample-fixture.js > /tests/test-results.txt
+```
+
+### -s \<path\>, --screenshots \<path\> 
+Enables screenshot capturing and specifies the directory path
+to save the screenshots to.
+
+```sh
+testcafe all /tests/sample-fixture.js -s /screenshots
+```
+
+### -S, --screenshots-on-fails
+Takes a screenshot whenever a test fails. Screenshots are saved to the directory
+specified by using the **-screenshots \<path\>** option.
+
+For example, the following command tells TestCafe to run tests from the
+  *sample-fixture.js* file in all browsers, take screenshots if tests fail
+  and save the screenshots to the */screenshots* directory.
+
+```sh
+testcafe all /tests/sample-fixture.js -S -s /screenshots
+```
+
+### -q, --quarantine-mode 
+Enables the quarantine mode for tests that fail. In this mode,
+a failed test runs five more times, and if it fails in all attempts,
+the test is marked as failed. If the test passes in any run,
+it is marked as unstable.
+
+```sh
+testcafe all /tests/sample-fixture.js -q
+```
+
+### -e, --skip-js-errors
+Makes tests not fail when a JavaScript error occurs on a tested web page.
+By default, in case of such errors, TestCafe stops test execution and posts
+an error message to a report. If you want TestCafe to ignore JavaScript errors,
+specify this argument.
+
+For example, the following command runs test from the specified file and tells TestCafe to ignore JavaScript errors.
+
+```sh
+testcafe ie /tests/sample-fixture.js -e
+```
+
+### -t \<name\>,  --test \<name\> 
+TestCafe will run a test with the specified name.
+
+For example, the following command runs only the *"Click a label"* test from the *sample-fixture.js* file.
+
+```sh
+testcafe ie /tests/sample-fixture.js -t "Click a label"
+```
+
+### -T \<pattern\>, --test-grep \<pattern\> 
+TestCafe will run tests whose names match the specified pattern. 
+
+For example, the following command runs the tests whose names match `Click*`. These can be the `Click a label`, `Click a button` tests and so on.
+
+```sh
+testcafe ie /tests -T Click*
+```
+
+### -f \<name\>, --fixture \<name\>
+TestCafe will run a fixture with the specified name.
+
+```sh
+testcafe ie /tests -f sample-fixture
+```
+
+### -F \<pattern\>, --fixture-grep \<pattern\>
+TestCafe will run fixtures whose names match the specified pattern.
+
+```sh
+testcafe ie /Tests --fixture-grep *Page*
+```
+
+### --ports \<port1,port2\>
+Specifies the custom port numbers used by TestCafe to perform testing. Number [0-65535].
+
+If ports are not specified, TestCafe picks ports by itself.
```

Comment:
*Style*:
To me, it offends the eye a little bit when you say *by itself* about software.